TicTacToeAppTests/BoardSpec.swift

```diff
@@ -0,0 +1,54 @@
+
+import Quick
+import Nimble
+import TicTacToeApp
+
+class BoardSpec: QuickSpec {
+  override func spec() {
+
+    describe("Board") {
```

Comment:
Let's plan on discussing best practices for test organization. Overall, I like the tests, however there are some tweaks that would make them a lot better.